arch/lkl/include/uapi/asm/syscalls.h

```diff
@@ -170,6 +170,8 @@ struct ustat {
 	char			f_fpack[6];
 };
 
+typedef __kernel_rwf_t		rwf_t;
```

Comment:
rwf_t looks like a kernel internal type, so I don't think we should not export it. What issue did you run into that this typedef fixes?

Reply:
I think ddef7ed2b5cbafae692d1d580bb5a07808926a9c requires this line

```
commit ddef7ed2b5cbafae692d1d580bb5a07808926a9c
Author: Christoph Hellwig <hch@lst.de>
Date:   Thu Jul 6 18:58:37 2017 +0200

    annotate RWF_... flags
    
    [AV: added missing annotations in syscalls.h/compat.h]
    
    Signed-off-by: Christoph Hellwig <hch@lst.de>
    Signed-off-by: Al Viro <viro@zeniv.linux.org.uk>
```

and `make` command fails with the errors.

```
  REPLACE       /home/tazaki/gitworks/lkl-linux-upstream/tools/lkl//include/lkl/linux/posix_types.h
  REPLACE       /home/tazaki/gitworks/lkl-linux-upstream/tools/lkl//include/lkl/linux/sembuf.h
  CC       /home/tazaki/gitworks/lkl-linux-upstream/tools/lkl/lklfuse.o
In file included from /home/tazaki/gitworks/lkl-linux-upstream/tools/lkl//include/lkl.h:13:0,   
                 from lklfuse.c:14:
/home/tazaki/gitworks/lkl-linux-upstream/tools/lkl//include/lkl/asm/syscall_defs.h:395:135: error: unknown type name ‘lkl_rwf_t’; did you mean ‘lkl_off_t’?
 LKL_SYSCALL_DEFINE6(_preadv2,unsigned long,fd,const struct lkl_iovec *,vec,unsigned long,vlen,unsigned long,pos_l,unsigned long,pos_h,lkl_rwf_t,flags)
                                                                                                                                       ^
/home/tazaki/gitworks/lkl-linux-upstream/tools/lkl//include/lkl/asm/syscalls.h:251:30: note: in definition of macro ‘__LKL__SC_DECL’
 #define __LKL__SC_DECL(t, a) t a
                              ^
(snip)
```

Reply:
Oh right, this is a kernel base type now, I got confused.